policyengine_us/variables/gov/states/la/tax/income/deductions/la_itemized_deductions.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class la_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = [
+        "https://revenue.louisiana.gov/TaxForms/IT540iWEB(2022)D1.pdf",
+        "https://www.legis.la.gov/Legis/Law.aspx?d=101760",  # (3)
+    ]
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.la.tax.income.deductions.itemized
+        itemizes = tax_unit("tax_unit_itemizes", period)
```

Comment:
we also add all itemized deductions (excluding SALT deduction and medical [which is added back]) [8A](https://revenue.louisiana.gov/TaxForms/IT540WEB(2022)%20F%20D2.pdf)